specification/billing/resource-manager/Microsoft.Billing/preview/2024-08-02-preview/enrollementToModernMigration.json

```diff
@@ -0,0 +1,245 @@
+            "name": "parameters",
+            "description": "The enrollment to modern mapping details.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/MigrationMappingProperties"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Transfer details.",
+            "schema": {
+              "$ref": "#/definitions/MigrateEnrollementResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "AssociatedTenantsListByBillingAccount": {
+            "$ref": "./examples/migrateEnrollement.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "MigrationMappingProperties": {
+      "description": "Enrollement to modern mapping properties.",
+      "type": "object",
+      "properties": {
+        "destinationBillingScope": {
+          "description": "Modern billing scope to which enrollment is to be mapped.",
+          "type": "string"
+        },
+        "migrateEnrollmentRequestStatus": {
+          "description": "Migration enrollement request status.",
+          "type": "string"
+        }
+      }
+    },
+    "GetEnrollementToModernMapResponse": {
+      "description": "Get enrollement to modern mapping details.",
+      "type": "object",
+       "properties": {
+        "id": {
+          "description": "Fully qualified ID for the migrateEntity resource.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "name": {
+          "description": "Resource name.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "type": {
+          "description": "Resource type, Microsoft.Billing/billingAccounts/migrationEntity.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "properties": {
+          "destinationBillingScope": {
+            "description": "Modern billing scope to which enrollment is to be mapped.",
+            "type": "string"
+          },
+          "enrollmentId": {
+            "description": "Enrollment Id",
+            "type": "string"
+          },
+          "status": {
+            "description": "Migration Status",
+            "type": "string"
+          },
+          "transferId": {
+            "description": "Transfer Id",
+            "type": "string"
+          }
+        }
+      }
+    },
+    "MigrateEnrollementResponse": {
+      "description": "Get enrollement to modern mapping details.",
+      "type": "object",
+       "properties": {
+        "id": {
+          "description": "Fully qualified ID for the migrateEntity resource.",
+          "readOnly": true,
+          "type": "string"
+        },
```

Comment:
If this is a fully qualified resource ID then please specify "arm-id" format.

Reply:
Deleted file from PR